pyqmc/linemin.py

```diff
@@ -178,14 +180,15 @@ def gradient_energy_function(x, coords):
         newparms = pgrad_acc.transform.deserialize(x)
         for k in newparms:
             wf.parameters[k] = newparms[k]
-        data, coords = vmc(wf, coords, accumulators={"pgrad": pgrad_acc}, **vmcoptions)
-        df = pd.DataFrame(data)[warmup:]
-        en = np.mean(df["pgradtotal"])
-        en_err = np.std(df["pgradtotal"]) / np.sqrt(len(df))
+        df, coords = vmc(wf, coords, accumulators={"pgrad": pgrad_acc}, client=client, npartitions=npartitions, **vmcoptions)
+        #df = pd.DataFrame(data)[warmup:]
```

Comment:
remove the commented line? or should we check if it's necessary to throw out some warmup steps? seems like if the parameters aren't changing much, the warmup isn't necessary